source/geod24/LocalRest.d

```diff
@@ -475,7 +475,9 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
                 }.format(member, ovrld.mangleof));
             }
         default:
-            assert(0, "Unmatched method name: " ~ cmd.method);
+            C.trySend(cmd.sender,
+                      Response(Status.Failed, cmd.id,
+                               SerializedData("Method not implemented (404)")));
```

Comment:
Would it be of any use to emit the method name in the message?

Reply:
Hmm actually it would make even more sense to throw a custom 404 exception. If vibe.d does something similar we could handle both in our agora codebase.